### PR TITLE
adds more templates for saying no on issues

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

I've had some requests for more templates for saying no on issues in different situations. This adds in more scenarios.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ x ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: communication/saying-no.md [hunk 1]
```diff
@@ -48,6 +48,32 @@ __What do I need to do?__
 If this issue is no longer important, no further action is necessary. However, if you think this is something that should be addressed, please open a new issue and refer to the original issue in the description.
 ```
 
+**Issue is not something we will ever do**
+
+```
+Thank you for taking time time to file this issue.
```
Comment (miah) [inline communication/saying-no.md]: time time

File: communication/saying-no.md [hunk 2]
```diff
@@ -48,6 +48,32 @@ __What do I need to do?__
...
+
+We understand and appreciate your reason for this request. However, at this time, it does not align with our core direction and is not something we’re looking at doing.
+
+Thank you again for filing this, however, we greatly appreciate your interest in this project!
+```
+
+**Issue is an interesting idea, be we do not have the resources right now**
```
Comment (miah) [inline communication/saying-no.md]: be we

File: communication/saying-no.md [hunk 3]
```diff
@@ -48,6 +48,32 @@ __What do I need to do?__
...
+
+```
+Thank you for taking time time to file this issue.
```
Comment (miah) [inline communication/saying-no.md]: time time

**Comment by nellshamrell** [comment]
Thanks for the comments, @miah - I fixed those typos!

**Pull request merged**